src/server/game/Globals/ConversationDataStore.cpp

```diff
@@ -0,0 +1,170 @@
+            conversationActor.Unk5 = fields[5].GetUInt32();
+        }
+        while (actorTemplates->NextRow());
+
+        TC_LOG_INFO("server.loading", ">> Loaded " SZFMTD " Conversation template actors in %u ms", _conversationActorTemplateStore.size(), GetMSTimeDiffToNow(oldMSTime));
+    }
+    else
+    {
+        TC_LOG_INFO("server.loading", ">> Loaded 0 Conversation template actors. DB table `conversation_actor_template` is empty.");
+    }
+
+    if (QueryResult lineTemplates = WorldDatabase.Query("SELECT Id, StartTime, UiCameraID, ActorIdx FROM conversation_line_template"))
+    {
+        uint32 oldMSTime = getMSTime();
+        uint32 count = 0;
+        do
+        {
+            Field* fields = lineTemplates->Fetch();
+
+            uint32 id = fields[0].GetUInt32();
+
+            if (!sConversationLineStore.LookupEntry(id))
+            {
+                TC_LOG_ERROR("sql.sql", "Table `conversation_line_template` has template for non existing ConversationLine (ID: %u), skipped", id);
+                continue;
+            }
+
+            ConversationLineTemplate& conversationLine = _conversationLineTemplateStore[id];
+            conversationLine.Id         = id;
+            conversationLine.StartTime  = fields[1].GetUInt32();
+            conversationLine.UiCameraID = fields[2].GetUInt32();
+            conversationLine.ActorIdx   = fields[3].GetUInt32();
+
+            ++count;
+        }
+        while (lineTemplates->NextRow());
+
+        TC_LOG_INFO("server.loading", ">> Loaded %u conversation template lines in %u ms", _conversationLineTemplateStore.size(), GetMSTimeDiffToNow(oldMSTime));
+    }
+    else
+    {
+        TC_LOG_INFO("server.loading", ">> Loaded 0 Conversation template lines. DB table `conversation_line_template` is empty.");
+    }
+
+    if (QueryResult templates = WorldDatabase.Query("SELECT Id, FirstLineId, LastLineEndTime, VerifiedBuild FROM conversation_template"))
+    {
+        uint32 oldMSTime = getMSTime();
+
+        do
+        {
+            Field* fields = templates->Fetch();
+
+            ConversationTemplate conversationTemplate;
+            conversationTemplate.Id                 = fields[0].GetUInt32();
+            conversationTemplate.FirstLineId        = fields[1].GetUInt32();
+            conversationTemplate.LastLineEndTime    = fields[2].GetUInt32();
+
+            if (QueryResult actors = WorldDatabase.PQuery("SELECT ConversationActorId FROM conversation_actors WHERE ConversationId = %u ORDER BY Idx", conversationTemplate.Id))
+            {
+                do
+                {
+                    Field* fields = actors->Fetch();
+                    uint32 actorId = fields[0].GetUInt32();
+                    if (ConversationActorTemplate const* conversationActorTemplate = Trinity::Containers::MapGetValuePtr(_conversationActorTemplateStore, actorId))
+                        conversationTemplate.Actors.push_back(*conversationActorTemplate);
```

Comment:
i don't see it too, i just copied stuff from ObjectMgr with few modifications, i'm going to change something about saving actors after i have move infos about them, do you have some? about the unknowns?